packages/mgt/src/components/mgt-people-picker/mgt-people-picker.scss

```diff
@@ -8,288 +8,338 @@
 @import '../../../../../node_modules/office-ui-fabric-core/dist/sass/References';
 @import '../../styles/shared-styles.scss';
 
+// common regardless of themes
 $avatar-margin: var(--avatar-margin, 0 4px 0 0);
 $avatar-size-s: var(--avatar-size-s, 24px);
 $avatar-size: var(--avatar-size, 48px);
 $initials-color: var(--initials-color, white);
 $initials-background-color: var(--initials-background-color, #{$ms-color-magenta});
-
-$input-border: var(--input-border, 2px solid #b3b0ad);
-$input-border-bottom: var(--input-border-bottom, $input-border);
-$input-border-right: var(--input-border-right, $input-border);
-$input-border-left: var(--input-border-left, $input-border);
-$input-border-top: var(--input-border-top, $input-border);
-
-$input-hover-color: var(--input-hover-color, #323130);
-$input-focus-color: var(--input-focus-color, #0078d4);
-$input-background-color: var(--input-background-color, var(--people-list-background-color, white));
-
-$selected-person-background-color: var(--selected-person-background-color, #f1f1f1);
-
-$dropdown-background-color: var(--dropdown-background-color, white);
-$dropdown-item-hover-background: var(--dropdown-item-hover-background, rgba(241, 241, 241, 0.87));
-
-$placeholder-default-color: var(--placeholder-default-color, #605e5c);
-$placeholder-focus-color: var(--placeholder-focus-color, #323130);
-$font-color: var(--font-color, #323130);
-
-:host {
-  border-radius: 2px;
-  font-family: var(--default-font-family, 'Segoe UI');
-  height: 38px;
-  display: contents;
-  height: 100%;
-  overflow: hidden;
+$placeholder: var(--placeholder-default-color, #605e5c);
+
+// variables with theme colors
+$dropdown-item__background-color: (
+  dark: var(--dropdown-item-hover-background, #333d47),
+  light: var(--dropdown-item-hover-background, rgba(241, 241, 241, 0.87))
+);
+
+$placeholder--focus: (
+  light: var(--placeholder-focus-color, #323130),
+  dark: var(--placeholder-focus-color, rgba(255, 255, 255, 0.8))
+);
+
+$dropdown__background-color: (
+  dark: var(--dropdown-background-color, #1f1f1f),
+  light: var(--dropdown-background-color, #{$white})
+);
+
+$bg: (
```

Comment:
I think like `$background` a little better. We use the full name in other places too.